docs/release-notes/1.3.0/release-1.3.0.md

```diff
@@ -0,0 +1,97 @@
+            string lossOperation = null,
+            string metricOperation = null,
+            string learningRateOperation = null,
+            float learningRate = 0.01f,
+            bool addBatchDimensionInput = false,
+            DnnFramework dnnFramework = DnnFramework.Tensorflow)
+
+  public static DnnEstimator ImageClassification(
+            this ModelOperationsCatalog catalog,
+            string featuresColumnName,
+            string labelColumnName,
+            string outputGraphPath = null,
+            string scoreColumnName = "Score",
+            string predictedLabelColumnName = "PredictedLabel",
+            string checkpointName = "_retrain_checkpoint",
+            Architecture arch = Architecture.InceptionV3,
+            DnnFramework dnnFramework = DnnFramework.Tensorflow,
+            int epoch = 10,
+            int batchSize = 20,
+            float learningRate = 0.01f,
+            bool measureTrainAccuracy = false)
+
+  ```
+
+  [Design specification](https://github.com/dotnet/machinelearning/blob/cd591dd492833964b6829e8bb2411fb81665ac6d/docs/specs/DNN/dnn_api_spec.md)
+
+  [Image classification (Inception V3) sample](https://github.com/dotnet/machinelearning/blob/b6146219156cb05e2e2cebd80fe4cda017f03470/docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/InceptionV3TransferLearning.cs))
+
+  [Image classification (Resnet V2 101) sample](https://github.com/dotnet/machinelearning/blob/b6146219156cb05e2e2cebd80fe4cda017f03470/docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearning.cs)
+
+- **PyTorch scoring support (PREVIEW)** ([#4054](https://github.com/dotnet/machinelearning/pull/4054))  
+  Introduces `Microsoft.ML.PyTorch` package that will enable scoring pytorch models in ML .NET. We plan to bring in PyTorch training next release.
+  ```cs
+  public static TorchModel LoadTorchModel(this ModelOperationsCatalog catalog, string modelLocation)
+  ```
+
+- **SQL Dataloader (PREVIEW)** ([#4035](https://github.com/dotnet/machinelearning/pull/4035))  
+  Introduces SQL Dataloader that enables training on SQL tables. This feature is also in early preview and customer feedback will be much appreciated. 
+
+  [Design specification](https://github.com/dotnet/machinelearning/pull/3857)
+
+  ```cs
+  public static DatabaseLoader CreateDatabaseLoader(this DataOperationsCatalog catalog,
+            params DatabaseLoader.Column[] columns)
+  ```    
+
+## **Bug Fixes**
+### Serious
+- **SaveOnnxCommand appears to ignore predictors when saving a model to ONNX format:** This broke export to ONNX functionality. ([3974][https://github.com/dotnet/machinelearning/issues/3974])
+
+- **Unable to use fasterrcnn onnx model** ([3963](https://github.com/dotnet/machinelearning/pull/3963))
+- **PredictedLabel is always true for Anomaly Detection:** This bug disabled scenarios like fraud detection using binary classification/PCA. ([#4039](https://github.com/dotnet/machinelearning/pull/4039))
```

Comment:
We recently created this Fraud Detection using the anomaly detection PCA trainer. We still need to test it with this bug fix coming in 1.3. 
This is the sample that could be highlighted in the RELEASE notes, if needed: https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/AnomalyDetection_CreditCardFraudDetection

Reply:
It does not make sense to include sample for a feature whose bug we fixed. This section is just highlighting the bugs we fixed.

Reply:
I have added this under sample's section